packages/docs/src/components/header/components/algolia-search/AlgoliaSearch.vue

```diff
@@ -27,30 +9,22 @@ import { Options, Vue } from 'vue-class-component'
   name: 'Search',
 })
 export default class Search extends Vue {
-  value = ''
-  color = 'gray'
-
-  onFocusHandler (colorValue: string, displayValue: string): void {
-    this.color = colorValue
-    const el: HTMLElement | null = document.querySelector('.ds-dropdown-menu')
-    if (el) { el.style.display = displayValue }
-  }
-
   initialize () {
     Promise.all([
-      // @ts-ignore
-      import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.js'),
-      // @ts-ignore
-      import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.css'),
-    ]).then(([docsearch]) => {
-      docsearch = docsearch.default
-      docsearch(Object.assign({
-        apiKey: 'be3528055c92da2ea5133b93ed548c6d',
+      import(/* webpackChunkName: "docsearch" */ '@docsearch/js'),
+    ]).then(([{ default: docsearch }]) => {
+      docsearch({
+        container: '#docsearch',
+        appId: 'DVNV64RN9R',
         indexName: 'vuestic',
-      }, {
-        inputSelector: '#algolia-search-input',
-        debug: true,
-      }))
+        apiKey: 'cd8e70cb466bf6df138543a38c33ea5e',
+        transformItems: process.env.NODE_ENV === 'development'
```

Comment:
Does `yarn serve:docs` work for you? 

I just get an error.

Reply:
It was wrong autocoplete, mut be fixed